src/browser/modules/User/UserList.test.tsx

```diff
@@ -29,8 +29,9 @@ import {
   listUsersQuery
 } from 'shared/modules/cypher/boltUserHelper'
 
-// Stubbing out title bar as it depends on store
+// Stubbing out title bar & template as they depend on store
 jest.mock('browser/modules/Frame/FrameTitlebar', () => () => null)
+jest.mock('browser/modules/Frame/FrameTemplate', () => () => null)
```

Comment:
Since you seem to have removed the connect() from the FrameTemplate, this isn't needed anymore?

Reply:
Yes but it renders FrameEditor which is connected. So either the template or the Editor needs to be stubbed.